cypress/integration/blocks-image.js

```diff
@@ -10,7 +10,7 @@ if (Cypress.env('API') !== 'guillotina') {
       cy.waitForResourceToLoad('@breadcrumbs');
       cy.waitForResourceToLoad('@actions');
       cy.waitForResourceToLoad('@types');
-      cy.waitForResourceToLoad('?fullobjects');
+      cy.waitForResourceToLoad('my-page?fullobjects');
```

Comment:
@sneridagh doesn't that wait for "/my-page/my-page?fullobjects"? I mean we are on "mypage/edit", right?

Reply:
I left it like this to be more explicit to know what are we expecting but later on, I realised the command is using an ‘includes’ so it does not make much of a difference.